iiop-openjdk/src/main/java/org/wildfly/iiop/openjdk/IIOPRootDefinition.java

```diff
@@ -57,7 +57,7 @@
  */
 class IIOPRootDefinition extends PersistentResourceDefinition {
 
-    static final RuntimeCapability<Void> IIOP_CAPABILITY = RuntimeCapability.Builder.of(Capabilities.IIOP_CAPABILITY, false).build();
+    public static final RuntimeCapability<Void> IIOP_CAPABILITY = RuntimeCapability.Builder.of(Capabilities.IIOP_CAPABILITY, false, Void.class).build();
```

Comment:
This service type conflicts with the type of the installed service.  What is the intention here?
It seems to me that this capability should provide an `org.omg.CORBA.ORB`, no?

Reply:
Also, does this need to be public?

Reply:
I changed this type to `CorbaORBService.class`. `CorbaORBService` implements `Service<ORB>`

Reply:
Capability service types should not leak the MSC API.

Reply:
I am not sure I get what "leak" means, Do you mean I should have used the service type `ORB` instead of `CorbaORBService` ?

Reply:
> Do you mean I should have used the service type ORB instead of CorbaORBService ?

That was my suggestion.

Reply:
Thanks, I have updated it to `ORB`

Reply:
The Consumer with which CorbaORBService is constructed still needs to be updated to reflect this.

Reply:
In that case the service consumer in `CorbaORBService` has to accept ORB instance`serviceConsumer.accept(this.orb);`, not the `CorbaORBService` instance. Is that correct ?

Reply:
Correct.

Reply:
I was just confused with the service type. I have updated the consumer within the `CorbaORBService`.

There are failures in `Galleon Linux - JDK 11` and `Windows - JDK 11`, I triggered another run for them. But I don't think they're related to this change.